kits/incident-postmortem-pipeline/.env.example

```
@@ -0,0 +1,4 @@
+LAMATIC_FLOW_ID="LAMATIC_FLOW_ID Flow ID"
+LAMATIC_API_URL="LAMATIC_API_URL"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)